feat: remove banner and update homepage

alisonjoseph opened the pull request:
Closes #1594
closes https://github.com/carbon-design-system/carbon-website/issues/1735
Ref https://github.com/carbon-design-system/carbon-website/issues/1424 (design spec)

Homepage design enhancements, including removing blue alert banner and adding a new tile.

#### Changelog

**New**

- homepage tile

**Changed**

- Updated homepage text.
- moved pause video button
- mobile/tablet styles for feature tile
- fix for height of side nav
- fix for anchor links

**Removed**

- Blue alert banner at top

## ToDo

- [x] Update feature tile images at different breakpoints

connor-leech commented on src/components/Homepage/Homepage.js:
```diff
@@ -2,39 +2,30 @@ import React from 'react';
+              <div className="bx--col-lg-4 bx--col-md-4 bx--col-sm-2 bx--offset-lg-12 bx--offset-md-4 bx--offset-sm-2 homepage--tile-header">
+                <ClickableTile
+                  title="Read"
+                  titleLarge="Migrate from v9 to v10"
```
```suggestion
                  titleLarge="Migrate to v10"
```

jeanservaas commented:
Hey @alisonjoseph thanks for picking this up!!

- At Xlg breakpoint, looks like the pause button is too far off the left menu right now, it should be 16px off the side nav and 16px up from the carbon design system band. basically it should just totally mimic the pause button positioning on the IDL site.

![image](https://user-images.githubusercontent.com/43144260/60465858-b5d00300-9c17-11e9-9439-e4d16d439b7b.png)

![image](https://user-images.githubusercontent.com/43144260/60465970-03e50680-9c18-11e9-96f5-1fd20db4f449.png)


- Also, in mobile, there shouldn't be a margin on the banner feature square

<br>

preview link

![image](https://user-images.githubusercontent.com/43144260/60469499-7e1a8880-9c22-11e9-81c8-0f2987cc7841.png)


<br>

spec
![image](https://user-images.githubusercontent.com/43144260/60469444-4d3a5380-9c22-11e9-8dd8-f70c7f38ce75.png)

<br>

- Feature tile should be the same size and aspect ratio as the start designing and start developing tiles -- right now feature tile looks smaller

<br>

preview link

![image](https://user-images.githubusercontent.com/43144260/60469610-fb45fd80-9c22-11e9-85ab-183bfb737ebe.png)

<br>

spec

![image](https://user-images.githubusercontent.com/43144260/60469643-1e70ad00-9c23-11e9-9603-947d968d9917.png)


<br>

- At tablet breakpoint the start designing and start developing tiles should only be 320px tall (i.e. have a 2:1 aspect ratio) -- they look much taller in the preview link for some reason and the >_ is getting really small and getting pushed off of vertical center. Also there should be 48px between the bottom of the Carbon Design System banner and the top of the Getting Started Design tile and 16px gutters (not 24px) between all images on the homepage


![image](https://user-images.githubusercontent.com/43144260/60470040-8247a580-9c24-11e9-8423-9ed3a128e23e.png)


- @shixiedesign brought this up on interior pages — need to adjust anchor links to account for missing blue bar (i.e. jog up position of anchor links 

![image](https://user-images.githubusercontent.com/43144260/60469058-1adc2680-9c21-11e9-97c8-31c5d6ad05f7.png)

alisonjoseph commented:
Quick question about this comment

> At tablet breakpoint the start designing and start developing tiles should only be 320px tall (i.e. have a 2:1 aspect ratio) 

If they are a set height, then they won't be a set aspect ratio because the width varies. They are a set height right now, but I think it is taller than 320px. I can adjust to just be 2:1 aspect ratio, I think thats what the spec is trying to say

alisonjoseph commented:
@jeanservaas I've made some updates.

- fixed video height at mobile breakpoint so it is 2x the square tile. 
- removed 1rem padding on left/right of square tile at mobile breakpoint
- updated spacing above/below the "getting started" tiles to be 48px/80px
- updated tablet sizing for "getting started" tiles to be set at 320px tall

tw15egan commented on src/styles/_sticky.scss:
```diff
@@ -89,6 +89,7 @@ website global header hidden
   // side nav visible, website alert not present
   .#{$prefix}--header ~ .#{$prefix}--side-nav {
     transform: translate(0, -3rem);
+    height: 100%;
```
Thank you for fixing this 🙏

alisonjoseph replied:
Yesss its been driving me crazy! 👍

alisonjoseph commented:
Don't merge until @jeanservaas approves pls 😄  not 100% sure if we have the tablet "getting started blocks" correct yet

jeanservaas commented:
Looking good, just a few more things!

- Just need to get the feature card and the start designing and start developing card the same size and dimensions at every breakpoint but mobile and also the same type tokens should be used (Migrate to v10 looks like a smaller type size at the Xlg breakpoint)

![image](https://user-images.githubusercontent.com/43144260/60530944-896fc180-9cbf-11e9-8a91-b3d845ab08a0.png)

- Also, can you change this margin (between start designing / start developing) to 16px from 24px?

![image](https://user-images.githubusercontent.com/43144260/60531555-d7d19000-9cc0-11e9-9c82-39bccfeb4439.png)


- Just noticed this but arrow colors and spacing also look different on feature card... banner feature card arrow looks too light (maybe it's using white instead of f3f3f3

![image](https://user-images.githubusercontent.com/43144260/60531748-3f87db00-9cc1-11e9-8733-c88c531ba127.png)

- Anchor link distance still seems to be happening (i.e. assuming the 40px blue banner is still there when you click to jump down the page. Again, this was a Shixie catch, so lmk if what I'm saying doesn't make sense here.

![image](https://user-images.githubusercontent.com/43144260/60531825-73fb9700-9cc1-11e9-98b4-db55d32e7cb8.png)


Thanks girl!!!

alisonjoseph commented:
- icons are all gray 10 
- card sizes should match now 
- larger title type is using the same tokens (productive heading 03)
- spacing between tiles is 16px/1rem
- updated anchor link offset from top (this one is a little tricky because some pages have tabs and others don't so I picked a value that works for both, we can look at updating based on whether a page has tabs or not in the theme or a future issue/PR)

Hopefully, this is it @jeanservaas 🙏
